packages/angular/package.json

```diff
@@ -7,51 +7,70 @@
     "dev": "ng build --watch",
     "start": "ng serve",
     "build": "ng build --prod",
-    "test": "echo \"Skipped: ng test\"",
+    "test": "jest",
     "lint": "ng lint"
   },
   "license": "Apache-2.0",
   "devDependencies": {
     "@angular-devkit/build-angular": "~0.1102.3",
     "@angular-eslint/builder": "4.3.0",
-    "@angular-eslint/eslint-plugin-template": "4.3.0",
     "@angular-eslint/eslint-plugin": "4.3.0",
+    "@angular-eslint/eslint-plugin-template": "4.3.0",
     "@angular-eslint/schematics": "4.3.0",
     "@angular-eslint/template-parser": "4.3.0",
     "@angular/animations": "~11.2.4",
     "@angular/cli": "~11.2.3",
     "@angular/common": "~11.2.4",
-    "@angular/compiler-cli": "~11.2.4",
     "@angular/compiler": "~11.2.4",
+    "@angular/compiler-cli": "~11.2.4",
     "@angular/core": "~11.2.4",
-    "@angular/platform-browser-dynamic": "~11.2.4",
+    "@angular/forms": "^11.2.0",
     "@angular/platform-browser": "~11.2.4",
+    "@angular/platform-browser-dynamic": "~11.2.4",
     "@angular/router": "~11.2.4",
-    "@types/jasmine": "~3.6.0",
+    "@types/jest": "^27.0.2",
     "@types/node": "^12.11.1",
     "@typescript-eslint/eslint-plugin": "4.16.1",
     "@typescript-eslint/parser": "4.16.1",
+    "canvas": "^2.9.3",
     "eslint": "^7.6.0",
-    "jasmine-core": "~3.6.0",
-    "jasmine-spec-reporter": "~5.0.0",
-    "karma-chrome-launcher": "~3.1.0",
-    "karma-coverage": "~2.0.3",
-    "karma-jasmine-html-reporter": "^1.5.0",
-    "karma-jasmine": "~4.0.0",
-    "karma": "~6.3.16",
+    "jest": "^27.2.1",
+    "jest-preset-angular": "^10.0.0",
     "ng-packagr": "^11.0.0",
     "protractor": "~7.0.0",
     "rxjs": "~6.6.0",
     "ts-node": "~8.3.0",
     "tslib": "^2.0.0",
     "typescript": "~4.1.5",
+    "ng-mocks": "^14.2.0",
     "zone.js": "~0.11.3"
   },
   "workspaces": {
     "nohoist": [
-      "@angular-devkit/build-angular",
-      "@types/jasmine"
+      "@angular-devkit/build-angular"
     ]
   },
-  "typings": "./dist/ui-angular/index.d.ts"
+  "typings": "./dist/ui-angular/index.d.ts",
+  "jest": {
+    "preset": "jest-preset-angular",
+    "setupFilesAfterEnv": [
+      "<rootDir>/projects/ui-angular/src/setup.jest.ts"
+    ],
+    "testPathIgnorePatterns": [
+      "<rootDir>/node_modules/",
+      "<rootDir>/dist/"
+    ],
+    "transformIgnorePatterns": [
+      "node_modules/(?!.*.(mjs|ts)$)"
+    ],
+    "transform": {
+      "^.+\\.(ts|js|mjs|html|svg)$": "jest-preset-angular"
+    },
+    "globals": {
+      "ts-jest": {
+        "tsconfig": "<rootDir>/tsconfig.spec.json",
+        "stringifyContentPathRegex": "\\.(html|svg)$"
+      }
+    }
+  }
```

Comment:
Can we move this in to a jest config file to follow the pattern set by other platforms?

Reply:
👍

Reply:
b226db3